languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -83988,6 +83988,29 @@ USA
             <example correction="must|have to">I <marker>must to</marker> go shopping.</example>
             <example>It is a must to be aware of that.</example>
         </rule>
+        <rule id="TO_AFTER_MODAL_VERBS" name="to after modal verbs" default="temp off">
```

Comment:
Hi @LeahKorol, the correct value here of the attribute `default` is `"temp_off"`, with the underscore.